src/gui/AstroCalcDialog.cpp

```diff
@@ -3225,24 +3230,19 @@ localSEparameter localSolarEclipse(double JD,int contact,bool central) {
 	double lat = static_cast<double>(core->getCurrentLocation().latitude);
 	double lon = static_cast<double>(core->getCurrentLocation().longitude);
 	double elevation = static_cast<double>(core->getCurrentLocation().altitude);
-	lat = lat * M_PI_180;
 	lon = -(lon);
```

Comment:
Not sure what this does. Is `lon` use further down? (It looks like in the previous solution it was required to switch from our eastern-positive location to an western-positive implementation?)

Else, very good that you can identify and clean up all these bits and pieces, thank you!

Reply:
Uhmm. I think we still need to do that.